indico/modules/rb_new/blueprint.py

```diff
@@ -31,6 +31,7 @@
 _bp.add_url_rule('/<path:path>', 'roombooking', RHRoomBooking)
 _bp.add_url_rule('/rooms-sprite-<version>.jpg', 'sprite', misc.RHRoomsSprite)
 _bp.add_url_rule('/rooms-sprite.jpg', 'sprite', misc.RHRoomsSprite)
+_bp.add_url_rule('/rooms-image/<int:room_id>/room.jpg', 'rooms_image', rooms.RHRoomsImage)
```

Comment:
What about `/rooms/<int:room_id>/photo.jpg` and then the RH in singular `RHRoomPhoto`? I don't have a strong opinion on Image vs Photo.
Also, this might fit better in # Rooms

Reply:
"Photo" seems a bit better. "Image" is more vague.

Reply:
I have no strong opinion on this. I could go with "photo".